---
title: Homepage
author: Eric Edward Bryant
---

<img src="/images/profile.jpg"
  width="200"
  height="200"
  alt="Logo"
  style="border-radius: 50%;
         display: block;
         margin-left: auto;
         margin-right: auto;">

<div style="text-align:center">
[<i class="fas fa-envelope"></i> Email](mailto:EricBryantPhD@gmail.com) /
[<i class="fab fa-github"></i> Github](https://ericbryantphd.com/github) /
[<i class="fab fa-linkedin"></i> LinkedIn](https://ericbryantphd.com/linkedin)\
[<i class="fab fa-researchgate"></i> ResearchGate](https://ericbryantphd.com/researchgate) /
[<i class="fab fa-orcid"></i> ORCID](https://ericbryantphd.com/orcid) /
</div>

-------------------------------------------------------------------------------

**Update 2023-02-25:** I am officially back on the job market! My CV is
available to read [here](cv), or download [here](cv#downloads).

Please [contact me](mailto:EricBryantPhD@gmail.com) if you are looking for
someone with an interest and expertise in any of the following areas:

- Genetic screening approaches, CRISPR systems & base editing
- R statistical programming, including package and shiny app development
- NGS methods, applications and computational workflows
- Systems genetics, interaction networks, synthetic lethality & epistasis
- DNA replication, recombination & repair
- Cancer genetics
- Yeast genetics
- Mammalian cell culture
- Industrial mammalian expression
- Biologics discovery and optimization

## About me

I am currently a postdoctoral research fellow at Amgen in Therapeutic Discovery,
where I am applying CIRSPR methods and transcriptomics to study the genetics
of complex molecule expression in mammalian cells. In this role I have gained
valuable hands-on experience in pooled and arrayed CRISPR screening laboratory
techniques, and computational methods, as well as extensive exposure to the
general pharmaceutical drug development pipeline.

Prior to my role at Amgen, I enjoyed a short-term contract with Roche as a 
clinical genomics data science consultant, where I evaluated the sensitivity of
variant calling from RNA sequencing data. Here I gained valuable experience with
NGS workflows and high performance computing environments in an industry
setting.

## Publication highlights

To read some of my PhD thesis work, check out the following publications:

- Rad5 dysregulation drives hyperactive recombination at replication forks
  resulting in cisplatin sensitivity and genome instability.\
  **Bryant\ EE**, Šunjevarić\ I, Berchowitz\ L, Rothstein\ R, Reid\ RJD.\
  _Nucleic Acids Research_. 2019 Sep 26;47(17):9144---9159\
  [PMID: 31350889](https://www.ncbi.nlm.nih.gov/pubmed/31350889)
  -- 
  [PMCID: PMC6753471](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6753471)
  -- 
  [DOI: 10.1093/nar/gkz631](https://doi.org/10.1093/nar/gkz631)


- CRISPR-mediated base editing enables efficient disruption of eukaryotic
  genes through induction of STOP codons.\
  Billon\ P\*, __Bryant\ EE\*__, Joseph\ SA, Nambiar\ TS, Hayward\ SB,
  Rothstein\ R, and Ciccia\ A.\
  _Molecular Cell_. 2017 Sep 21;67(6):1068---1079.e4\
  [PMID: 28890334](https://www.ncbi.nlm.nih.gov/pubmed/28890334)
  -- 
  [PMCID: PMC5610906](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610906)
  -- 
  [DOI: 10.1016/j.molcel.2017.08.008](https://doi.org/10.1016/j.molcel.2017.08.008)\
  ^__\*co-first authors__^

Want to know more about my PhD work? Then check out my post on the
[systems genetics of DNA damage tolerance](2020/01/09/phd-dissertation), 
which includes,
[informal introductions to the topics covered in my dissertation](2020/01/09/phd-dissertation#preface),
and
[discussion of key insights from my PhD](2020/01/09/phd-dissertation#discussion).
A complete list of my publications is available [here](cv#publications).

## Software highlights

For examples of research software I have developed, check out the following:

2019
:   **[screenmill R package](https://github.com/ericedwardbryant/screenmill):
    capture, annotate, quantify, review and analyze time-series colony
    growth.**
    This package was used for colony quantification and interaction analysis
    in Bryant et al. [2019](https://doi.org/10.1093/nar/gkz631).
    A nice example analysis using `screenmill` can be found in
    [Figure 1D](https://academic.oup.com/view-large/figure/163637130/gkz631fig1.jpg).
    A shiny app is included to enable manual review of processed images.

2017
:   **[iSTOP R package](https://github.com/ericedwardbryant/iSTOP): design
    guides to introduce stop codons with CRISPR-mediated base editors.**
    This package was written to facilitate guide design for
    Billon et el. [2017](https://doi.org/10.1016/j.molcel.2017.08.008).
    `iSTOP` can be configured to design guides to generate any desired missense
    mutation using any hypothetical base editor in any genome with
    annotated coding sequence coordinates.
