---
title: Homepage
author: Eric Edward Bryant
---



<p><img src="/images/profile.jpg"
  width="200"
  height="200"
  alt="Logo"
  style="border-radius: 50%;
         display: block;
         margin-left: auto;
         margin-right: auto;"></p>
<div style="text-align:center">
<p><a href="mailto:EricBryantPhD@gmail.com"><i class="fas fa-envelope"></i> Email</a> /
<a href="https://ericbryantphd.com/github"><i class="fab fa-github"></i> Github</a> /
<a href="https://ericbryantphd.com/linkedin"><i class="fab fa-linkedin"></i> LinkedIn</a><br />
<a href="https://ericbryantphd.com/researchgate"><i class="fab fa-researchgate"></i> ResearchGate</a> /
<a href="https://ericbryantphd.com/orcid"><i class="fab fa-orcid"></i> ORCID</a> /</p>
</div>
<hr />
<p><strong>Update 2023-02-25:</strong> I am officially back on the job market! My CV is
available to read <a href="cv">here</a>, or download <a href="cv#downloads">here</a>.</p>
<p>Please <a href="mailto:EricBryantPhD@gmail.com">contact me</a> if you are looking for
someone with an interest and expertise in any of the following areas:</p>
<ul>
<li>Genetic screening approaches, CRISPR systems &amp; base editing</li>
<li>R statistical programming, including package and shiny app development</li>
<li>NGS methods, applications and computational workflows</li>
<li>Systems genetics, interaction networks, synthetic lethality &amp; epistasis</li>
<li>DNA replication, recombination &amp; repair</li>
<li>Cancer genetics</li>
<li>Yeast genetics</li>
<li>Mammalian cell culture</li>
<li>Industrial mammalian expression</li>
<li>Biologics discovery and optimization</li>
</ul>
<div id="about-me" class="section level2">
<h2>About me</h2>
<div id="after-the-phd" class="section level3">
<h3>After the PhD</h3>
<p>I am currently a postdoctoral research fellow at Amgen in Therapeutic Discovery,
where I am applying CIRSPR methods and transcriptomics to study the genetics
of complex molecule expression in mammalian cells. In this role I have gained
valuable hands-on experience in pooled and arrayed CRISPR screening laboratory
techniques, and computational methods, as well as extensive exposure to the
general pharmaceutical drug development pipeline.</p>
<p>Prior to my role at Amgen, I enjoyed a short-term contract with Roche as a
clinical genomics data science consultant, where I evaluated the sensitivity of
variant calling from RNA sequencing data. Here I gained valuable experience with
NGS workflows and high performance computing environments in an industry
setting.</p>
</div>
<div id="the-phd" class="section level3">
<h3>The PhD</h3>
<p>My PhD was completed in 2018 at Columbia University under the mentorship of
Professor Rodney Rothstein and co-mentorship by Professor Alberto Ciccia.
To read some of my PhD thesis work, check out the following publications:</p>
<ul>
<li><p>Rad5 dysregulation drives hyperactive recombination at replication forks
resulting in cisplatin sensitivity and genome instability.<br />
<strong>Bryant EE</strong>, Šunjevarić I, Berchowitz L, Rothstein R, Reid RJD.<br />
<em>Nucleic Acids Research</em>. 2019 Sep 26;47(17):9144—9159<br />
<a href="https://www.ncbi.nlm.nih.gov/pubmed/31350889">PMID: 31350889</a>
–
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6753471">PMCID: PMC6753471</a>
–
<a href="https://doi.org/10.1093/nar/gkz631">DOI: 10.1093/nar/gkz631</a></p></li>
<li><p>CRISPR-mediated base editing enables efficient disruption of eukaryotic
genes through induction of STOP codons.<br />
Billon P*, <strong>Bryant EE*</strong>, Joseph SA, Nambiar TS, Hayward SB,
Rothstein R, and Ciccia A.<br />
<em>Molecular Cell</em>. 2017 Sep 21;67(6):1068—1079.e4<br />
<a href="https://www.ncbi.nlm.nih.gov/pubmed/28890334">PMID: 28890334</a>
–
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610906">PMCID: PMC5610906</a>
–
<a href="https://doi.org/10.1016/j.molcel.2017.08.008">DOI: 10.1016/j.molcel.2017.08.008</a><br />
<sup><strong>*co-first authors</strong></sup></p></li>
</ul>
<p>Want to know more about my PhD work? Then check out my post on the
<a href="2020/01/09/phd-dissertation">systems genetics of DNA damage tolerance</a>,
which includes,
<a href="2020/01/09/phd-dissertation#preface">informal introductions to the topics covered in my dissertation</a>,
and
<a href="2020/01/09/phd-dissertation#discussion">discussion of key insights from my PhD</a>.
A complete list of my publications is also available <a href="cv#publications">here</a>.</p>
</div>
</div>
<div id="software-highlights" class="section level2">
<h2>Software highlights</h2>
<p>For examples of my research software, check out the following:</p>
<ul>
<li><p><strong><a href="https://github.com/ericedwardbryant/screenmill">screenmill R package</a>:
capture, annotate, quantify, review and analyze time-series colony growth.</strong>
This package was used for colony quantification and interaction analysis in
<a href="https://doi.org/10.1093/nar/gkz631">Bryant et al., <em>NAR</em> 2019</a>.
A nice example analysis using <code>screenmill</code> can be found in
<a href="https://academic.oup.com/view-large/figure/163637130/gkz631fig1.jpg">Figure 1D</a>.
A shiny app is included to enable manual review of processed images.</p></li>
<li><p><strong><a href="https://github.com/ericedwardbryant/iSTOP">iSTOP R package</a>: design
guides to introduce stop codons with CRISPR-mediated base editors.</strong>
This package was written to facilitate guide design for
<a href="https://doi.org/10.1016/j.molcel.2017.08.008">Billon et al., <em>Mol Cell</em> 2017</a>.
<code>iSTOP</code> can be configured to design guides to generate any desired missense
mutation using any hypothetical base editor in any genome with annotated
coding sequence coordinates.</p></li>
</ul>
<p>These packages were written largely for personal use to enable analyses for
the publications during my PhD that I highlight above. They were never submitted
to CRAN, or Bioconductor repositories as they lack appropriate test coverage,
and were part of my R package development learning journey. That said, I am
proud to say that my previous collaborators still use these tools for ongoing
work, including to design the CRISPR base editing guide library in
<a href="https://doi.org/10.1016/j.cell.2021.01.041">Cuella-Martin et al., <em>Cell</em> 2021</a>.
Pretty neat to enable such exciting work and to get an acknowledgement in Cell!</p>
</div>
