---
title: Curriculum vitae
author: Eric Edward Bryant
output:
  blogdown::html_page:
    toc: true
---

```{r include = FALSE}
knitr::opts_knit$set(root.dir = here::here())
```

[Systems genetics of DNA damage tolerance --- cisplatin, _RAD5_ & CRISPR-mediated nonsense]: https://academiccommons.columbia.edu/doi/10.7916/d8-k1d0-kb09

[Elizabeth Miller]: https://www2.mrc-lmb.cam.ac.uk/group-leaders/h-to-m/liz-miller
[Songtao Jia]: https://biology.columbia.edu/people/jia
[Matthew Weitzman]: http://pathology.med.upenn.edu/department/people/517/matthew-d-weitzman
[Alberto Ciccia]: https://www.ciccialab.com
[Rodney Rothstein]: https://systemsbiology.columbia.edu/faculty/rodney-rothstein
[Ron Prywes]: https://biology.columbia.edu/people/prywes
[Chloë Bulinski]: https://biology.columbia.edu/people/bulinski
[Shannon Phan]: https://www.linkedin.com/in/shannon-phan-5058098
[Vitalay Fomin]: https://www.linkedin.com/in/vitalayfomin
[Venus So]: https://www.linkedin.com/in/venusso
[Irwin Chen]: https://www.linkedin.com/in/irwin-chen-2667ba2
[Rene Hubert]: https://www.linkedin.com/in/rene-hubert-82ab27143

[GSAS Biological Sciences]: https://biology.columbia.edu
[UCLA Microbiology, Immunology & Molecular Genetics]: https://www.mimg.ucla.edu
[WVC Biology]: https://www.westvalley.edu/academics/biology/
[New York Academy of Sciences: Genome Integrity Discussion Group]: https://www.nyas.org/programs/frontiers-of-science/genome-integrity-discussion-group/

[Genetics Society of America]: https://genetics-gsa.org
[NOLS Alumni]: https://www.nols.edu


-------------------------------------------------------------------------------


# <i class="fas fa-address-card"></i> Contact


| Eric Edward Bryant, PhD
[<i class="fas fa-envelope"></i> Email](mailto:EricBryantPhD@gmail.com) /
[<i class="fab fa-github"></i> Github](https://ericedwardbryant.netlify.app/github)\
[<i class="fab fa-researchgate"></i> ResearchGate](https://ericedwardbryant.netlify.app/researchgate) /
[<i class="fab fa-orcid"></i> ORCID](https://ericedwardbryant.netlify.app/orcid) /
[<i class="fab fa-linkedin"></i> LinkedIn](https://ericedwardbryant.netlify.app/linkedin)


-------------------------------------------------------------------------------


# <i class="fas fa-briefcase"></i> Employment

2020---Present
:   | **Amgen, Postdoctoral Research Fellow**
    | Developed and executed a research plan studying the genetics of mammalian
      recombinant protein expression using CRISPR methods and transcriptomics.
    | _Managers:_   [Irwin Chen] & [Rene Hubert]
    | _Keywords:_   <small>
                    High-throughput Mammalian Expression / RNAseq /
                    Arrayed & pooled CRISPR screening / Protein Engineering /
                    Multispecifics / Biologics Discovery / Biologics
                    Optimization
                    </small>

2020 Spring
:   | **Roche/Genentech, Clinical Genomics Data Science Consultant**
    | Evaluated variant calling from RNAseq while on a short-term consulting
      contract with pREDi (Pharma Research & Early Development Informatics).
    | _Managers:_   [Venus So] & [Vitalay Fomin]
    | _Keywords:_   <small>
                    Genome Analysis toolkit (GATK) / RNAseq / Real World Data /
                    High-performance Computing Cluster (HPC) / Bioinformatics
                    Workflow Managers, CWL, Nextflow, Seven Bridges Genomics
                    </small>

2009---2011
:   | **Gene Oracle, Research Assistant**
    | _Role:_ De-novo gene synthesis & degenerate codon library construction. 
      Pipeline management and development.
    | _Manager:_ [Shannon Phan]


-------------------------------------------------------------------------------


# <i class="fas fa-graduation-cap"></i> Education

2011---2018
:   | **PhD, Columbia University**
    | _Department:_ [GSAS Biological Sciences]
    | _Thesis:_     [Systems genetics of DNA damage tolerance --- cisplatin,
                    _RAD5_ & CRISPR-mediated nonsense]
    | _Mentor:_     [Rodney Rothstein]
    | _Co-Mentor:_  [Alberto Ciccia]
    | _Committee:_  [Songtao Jia], [Elizabeth Miller] & [Matthew Weitzman]
    | _Keywords:_   <small>
                    Systems genetics / Genetic interaction networks /
                    Landscape enrichment analysis / Synthetic lethality /
                    DNA replication, recombination & repair / 
                    Chromosome mobility / CRISPR iSTOP / Base editing / 
                    Bioinformatics / R statistical programming, package &
                    shiny app development
                    </small>

2006---2009
:   | **BS, University of California Los Angeles**
    | _Department:_ [UCLA Microbiology, Immunology & Molecular Genetics]

2004---2006
:   | **West Valley College**
    | _Department:_ [WVC Biology]


-------------------------------------------------------------------------------


# <i class="fab fa-r-project"></i> Software

2019
:   **[screenmill R package](https://github.com/ericedwardbryant/screenmill):
    capture, annotate, quantify, review and analyze time-series colony
    growth.**
    This package was used for colony quantification and interaction analysis
    in [Bryant et al., _NAR_ 2019](https://doi.org/10.1093/nar/gkz631).
    A nice example analysis using `screenmill` can be found in
    [Figure 1D](https://academic.oup.com/view-large/figure/163637130/gkz631fig1.jpg).
    A shiny app is included to enable manual review of processed images.

2017
:   **[iSTOP R package](https://github.com/ericedwardbryant/iSTOP): design
    guides to introduce stop codons with CRISPR-mediated base editors.**
    This package was written to facilitate guide design for
    [Billon et al., _Mol Cell_ 2017](https://doi.org/10.1016/j.molcel.2017.08.008).
    `iSTOP` can be configured to design guides to generate any desired missense
    mutation using any hypothetical base editor in any genome with
    annotated coding sequence coordinates.


-------------------------------------------------------------------------------


# <i class="fas fa-chalkboard-teacher"></i> Teaching

2013
:   | **Columbia University, Teaching Assistant for Molecular Biology**
    | _Professors:_ [Songtao Jia] & [Ron Prywes]

2012
:   | **Columbia University, Teaching Assistant for Cell Biology**
    | _Professors:_ [Elizabeth Miller] & [Chloë Bulinski]


-------------------------------------------------------------------------------


# <i class="fas fa-trophy"></i> Awards


2018
:   Departmental distinction for PhD dissertation defense

2017---2018
:   TL1 NIH training grant, clinical and translational research

2016---2017
:   T32 NIH training grant, cancer biology

2013---2015
:   T32 NIH training grant, biological sciences

2013
:   James Howard McGregor award
    (student with unusual promise as a teacher of zoology)


-------------------------------------------------------------------------------


# <i class="fas fa-users"></i> Member

2012---2018
:   [New York Academy of Sciences: Genome Integrity Discussion Group]

2012---2018
:   [Genetics Society of America]

2006---present
:   [NOLS Alumni]


-------------------------------------------------------------------------------


# <i class="fas fa-book-open"></i> Publications

#### Lead contribution

2019-09-26
:   Rad5 dysregulation drives hyperactive recombination at replication forks
    resulting in cisplatin sensitivity and genome instability.\
    **Bryant\ EE**, Šunjevarić\ I, Berchowitz\ L, Rothstein\ R, Reid\ RJD.\
    _Nucleic Acids Research_. 2019 Sep 26;47(17):9144---9159\
    [PMID: 31350889](https://www.ncbi.nlm.nih.gov/pubmed/31350889)
    -- 
    [PMCID: PMC6753471](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6753471)
    -- 
    [DOI: 10.1093/nar/gkz631](https://doi.org/10.1093/nar/gkz631)

2019-01-09
:   Systems genetics of DNA damage tolerance -- Cisplatin, _RAD5_ & 
    CRISPR-mediated nonsense.\
    **Bryant\ EE**.\
    _Columbia University_.\
    [DOI: 10.7916/d8-k1d0-kb09](https://doi.org/10.7916/d8-k1d0-kb09)

2017-09-21
:   CRISPR-mediated base editing enables efficient disruption of eukaryotic
    genes through induction of STOP codons.\
    Billon\ P\*, __Bryant\ EE\*__, Joseph\ SA, Nambiar\ TS, Hayward\ SB,
    Rothstein\ R, and Ciccia\ A.\
    _Molecular Cell_. 2017 Sep 21;67(6):1068---1079.e4\
    [PMID: 28890334](https://www.ncbi.nlm.nih.gov/pubmed/28890334)
    -- 
    [PMCID: PMC5610906](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610906)
    -- 
    [DOI: 10.1016/j.molcel.2017.08.008](https://doi.org/10.1016/j.molcel.2017.08.008)\
    ^__\*co-first authors__^


#### Supporting contribution

2022-03-24
:   Temporal coordination between chromosome mobility and homologous recombination.\
    Joseph\ F, Lee\ SJ, **Bryant\ EE**, Reid\ RJD, Šunjevarić\ I, Rothstein\ R.\
    _BioRxiv_. 2022.\
    [BioRxiv: 10.1101/2022.03.24.485580v1](https://www.biorxiv.org/content/10.1101/2022.03.24.485580v1)
    --
    [DOI: 10.1101/2022.03.24.485580](https://doi.org/10.1101/2022.03.24.485580)

2020-08-26
:   Measuring chromosome pairing during homologous recombination in yeast.\
    Joseph\ F, Lee\ SJ, **Bryant\ EE**, Rothstein\ R.\
    _Methods in Molecular Biology_. 2021;2153:253---265.\
    [PMID: 32840785](https://www.ncbi.nlm.nih.gov/pubmed/32840785)
    --
    [DOI: 10.1007/978-1-0716-0644-5_18](https://doi.org/10.1007/978-1-0716-0644-5_18)

2019-10-01
:   DNA damage triggers increased mobility of chromosomes in G1 phase cells.\
    Smith\ MJ, **Bryant\ EE**, Joseph\ FJ, Rothstein\ R.\
    _Molecular Biology of the Cell_. 2019 Oct 1;30(21):2620---2625\
    [PMID: 31483739](https://www.ncbi.nlm.nih.gov/pubmed/31483739)
    --
    [PMCID: PMC6761769](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6761769)
    --
    [DOI: 10.1091/mbc.E19-08-0469](https://doi.org/10.1091/mbc.E19-08-0469)

2018-09-01
:   Increased chromosomal mobility after DNA damage is controlled by
    interactions between the recombination machinery and the checkpoint.\
    Smith\ MJ, **Bryant\ EE**, Rothstein\ R.\
    _Genes & Development_. 2018 Sep 1;32(17-18):1242---1251\
    [PMID: 30181361](https://www.ncbi.nlm.nih.gov/pubmed/30181361)
    --
    [PMCID: PMC6120718](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6120718)
    --
    [DOI: 10.1101/gad.317966.118](https://doi.org/10.1101/gad.317966.118)

2016-10-01
:   A synthetic dosage lethal genetic interaction between CKS1B and PLK1 is
    conserved in yeast and human cancer cells.\
    Reid\ RJD, Du\ X, Šunjevarić\ I, Rayannavar\ V, Dittmar\ J,
    **Bryant\ EE**, Maurer\ M, and Rothstein\ R.\
    _Genetics_. 2016 Oct 1;204(2):807---819\
    [PMID: 27558135](https://www.ncbi.nlm.nih.gov/pubmed/27558135)
    --
    [PMCID: PMC5068864](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5068864)
    --
    [DOI: 10.1534/genetics.116.190231](https://doi.org/10.1534/genetics.116.190231)


#### Select Acknowledgements

2021-02-18
:   Functional interrogation of DNA damage response variants with base editing
    screens.\
    Cuella-Martin\ R, Hayward\ SB, Fan\ X, Chen\ Xiao, Huang\ JW,
    Taglialatela\ A, Leuzzi\ G, Zhao\ J, Rabadan\ R, Lu\ C, Shen\ Y, Ciccia A.\
    _Cell_. 2021 Feb 18;184(4):1081-1097.e19\
    [PMID: 33606978](https://www.ncbi.nlm.nih.gov/pubmed/33606978)
    --
    [PMCID: PMC8018281](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8018281)
    --
    [DOI: 10.1016/j.cell.2021.01.041](https://doi.org/10.1016/j.cell.2021.01.041)\
    **Acknowledged for bioinformatics support. I designed the CRISPR base
    editing guide library.**

-------------------------------------------------------------------------------


# <i class="fas fa-file-download"></i> Downloads

```{r echo = FALSE}
xfun::embed_file(
  "static/documents/CV-Eric-Edward-Bryant-PhD.pdf",
  name = "CV-Eric-Edward-Bryant-PhD.pdf",
  text = 'CV (.pdf)'
)
```
